tests/system/webdriver/tests/services/ConfigFrontEnd0002Test.php

```diff
@@ -76,8 +76,11 @@ public function tearDown()
 	 */
 	public function testChangeMetaDescription()
 	{
-		$this->siteHomePage->changeMetaDescription('JoomlaTestMetaDescription');
+		$this->assertEquals($this->previousMetaDescription, $this->siteHomePage->getMetaDescription(), 'Meta Description not changed');
 
-		$this->assertNotEquals('JoomlaTestMetaDescription', $this->siteHomePage->getMetaDescription(), 'Site Meta Description has not changed');
+		$newMetaDescription = 'JoomlaTestMetaDescription' . rand(1,100);
+		$this->siteHomePage->changeMetaDescription($newMetaDescription);
+
+		$this->assertNotEquals($this->previousMetaDescription, $this->siteHomePage->getMetaDescription(), 'Site Meta Description has not changed');
```

Comment:
Assertion Text needs to be changed, It should say
'Meta Description Should Change'